clickhouse-http-client/src/test/java/com/clickhouse/client/http/ClickHouseHttpClientTest.java

```diff
@@ -257,8 +255,8 @@ public void testTransaction() throws ClickHouseException {
         testImplicitTransaction();
     }
 
-    @Test // (groups = "integration")
-    public void testSslClientAuth() throws ClickHouseException {
+    @Test(groups = {"integration"})
+    public void testSslRootCertificateClientAuth() throws ClickHouseException {
```

Comment:
Can we promote the test to `ClickHouseClientTest` so that we can test `clickhouse-cli-client` as well? We can skip the test in `clickhouse-grpc-client` since it will be dropped in the near future.

Reply:
@zhicwu I am also thinking of dropping the ```clickhouse-cli-client```
I think it's OK to run it only in this scoop for now.

Reply:
Sure, let's mark this as resolved.